### DISCUSSION TITLE
Mysterious circle on ND

### DISCUSSION THREAD

**Discussion by CptBayer** [flybywiresim/aircraft#7192] [General]
Hi developers,

just for info. Mysterious circleon ND ...

![grafik](https://user-images.githubusercontent.com/76531893/168264099-348afe51-2cf1-4ab0-9b39-54bf98dd36ea.png)

greets from germany

**Comment by tracernz** [comment]
How does it fly? Looks like it might be a drawing issue rather than a path issue...

**Comment by CptBayer** [comment]
I didn´t flew the path cause I used a DIR prior reaching waypoint BERGO to get rid of it :-p ...

**Comment by emsss78** [comment]
I had a similar circle in one of my flight plans recently...I thought it was a drawing issue
